src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -543,14 +543,26 @@ public static void Atan2<T>(T y, System.ReadOnlySpan<T> x, System.Span<T> destin
         public static void BitwiseAnd<T>(System.ReadOnlySpan<T> x, System.ReadOnlySpan<T> y, System.Span<T> destination) where T : System.Numerics.IBitwiseOperators<T, T, T> { }
         public static void BitwiseAnd<T>(System.ReadOnlySpan<T> x, T y, System.Span<T> destination) where T : System.Numerics.IBitwiseOperators<T, T, T> { }
         public static void BitwiseOr<T>(System.ReadOnlySpan<T> x, System.ReadOnlySpan<T> y, System.Span<T> destination) where T : System.Numerics.IBitwiseOperators<T, T, T> { }
         public static void BitwiseOr<T>(System.ReadOnlySpan<T> x, T y, System.Span<T> destination) where T : System.Numerics.IBitwiseOperators<T, T, T> { }
         public static void Cbrt<T>(System.ReadOnlySpan<T> x, System.Span<T> destination) where T : System.Numerics.IRootFunctions<T> { }
         public static void Ceiling<T>(System.ReadOnlySpan<T> x, System.Span<T> destination) where T : System.Numerics.IFloatingPoint<T> { }
+        public static void Clamp<T>(System.ReadOnlySpan<T> x, System.ReadOnlySpan<T> min, System.ReadOnlySpan<T> max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
+        public static void Clamp<T>(System.ReadOnlySpan<T> x, System.ReadOnlySpan<T> min, T max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
+        public static void Clamp<T>(System.ReadOnlySpan<T> x, T min, System.ReadOnlySpan<T> max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
+        public static void Clamp<T>(System.ReadOnlySpan<T> x, T min, T max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
+        public static void Clamp<T>(T x, System.ReadOnlySpan<T> min, System.ReadOnlySpan<T> max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
+        public static void Clamp<T>(T x, System.ReadOnlySpan<T> min, T max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
+        public static void Clamp<T>(T x, T min, System.ReadOnlySpan<T> max, System.Span<T> destination) where T : System.Numerics.INumber<T> { }
         public static void ConvertChecked<TFrom, TTo>(System.ReadOnlySpan<TFrom> source, System.Span<TTo> destination) where TFrom : System.Numerics.INumberBase<TFrom> where TTo : System.Numerics.INumberBase<TTo> { }
         public static void ConvertSaturating<TFrom, TTo>(System.ReadOnlySpan<TFrom> source, System.Span<TTo> destination) where TFrom : System.Numerics.INumberBase<TFrom> where TTo : System.Numerics.INumberBase<TTo> { }
+        public static void ConvertToIntegerNative<TFrom, TTo>(System.ReadOnlySpan<TFrom> source, System.Span<TTo> destination) where TFrom : System.Numerics.IFloatingPoint<TFrom> where TTo : System.Numerics.IBinaryInteger<TTo> { }
+        public static void ConvertToInteger<TFrom, TTo>(System.ReadOnlySpan<TFrom> source, System.Span<TTo> destination) where TFrom : System.Numerics.IFloatingPoint<TFrom> where TTo : System.Numerics.IBinaryInteger<TTo> { }
```

Comment:
The API design in https://github.com/dotnet/runtime/issues/103611 has these listed as returning spans. That does not follow the pattern we've used for the rest of TensorPrimitives and I assumed it was a copy/paste error, so I have these returning void.